docs/source/Learn/makingModules/cppModules/cppModules-1.rst

```diff
@@ -69,20 +78,35 @@ Be sure to add descriptions to both the module and to the class variables used.
 
 Required Module Methods
 -----------------------
-Each module should define the ``Reset()`` method that is called when initializing the BSK simulation, and the ``UpdateState()`` method which is called every time the task to which the module is added is updated.
+Each module should define the ``Reset()`` method that is called when initializing the BSK simulation,
+and the ``UpdateState()`` method which is called every time the task to which the module is added is updated.
 
 Module Variables
 ----------------
-The public class variables can contain any variables that you want to have access to from the python layer.  For example, you might want to set a gain variable, the spacecraft area for solar radiation pressure evaluation, etc.
+The user configurable module variables should be private variables that are set and retrieved
```

Comment:
I'm going to update this description to be more general and say the variables should be private or protected.  The latter makes sense if the module is a sub-class of a parent module and should have ready access to these module variables.  I've already run into this in another branch.

Reply:
I pushed an updated commit that discusses when using `protected' makes sense.